src/lib/services/setting-service.ts

```diff
@@ -49,15 +49,15 @@ export default class SettingService {
             await this.eventService.storeEvent(
                 new SettingUpdatedEvent({
                     createdBy,
-                    data: { id },
+                    data: { id, ...value },
```

Comment:
Are we guaranteed that value is an object? Can it ever be undefined?

Reply:
The message signature is `async insert(id: string, value: object, createdBy: string):` afaik that only accepts objects?